evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -170,14 +182,15 @@ def test_make_pipeline(
                 )
             else:
                 expected_components = (
-                    label_encoder
+                    dfs
+                    + label_encoder
                     + replace_null
                     + email_featurizer
                     + url_featurizer
                     + drop_null
                     + delayed_features
-                    + natural_language_featurizer
                     + datetime
+                    + natural_language_featurizer
```

Comment:
I'm not sure how this was passing before. For non-TS problems the `NaturalLanguageFeaturizer` should always go after the `DatetimeFeaturizer`